include/geode/mesh/core/grid.h

```diff
@@ -98,65 +71,28 @@ namespace geode
             return 1 << dimension;
         }
 
-        local_index_t nb_cell_facets() const
-        {
-            return nb_cell_facets_static();
-        }
-
-        static constexpr local_index_t nb_cell_facets_static()
-        {
-            return 2 * dimension;
-        }
-
-        index_t nb_cells() const;
-
-        index_t nb_cells_in_direction( index_t direction ) const;
-
         double cell_length_in_direction( index_t direction ) const;
 
         double cell_size() const;
 
-        virtual index_t cell_index(
-            const GridCellIndices< dimension >& index ) const = 0;
-
-        virtual GridCellIndices< dimension > cell_indices(
-            index_t index ) const = 0;
-
-        absl::optional< GridCellIndices< dimension > > next_cell(
-            const GridCellIndices< dimension >& index,
-            index_t direction ) const;
-
-        absl::optional< GridCellIndices< dimension > > previous_cell(
-            const GridCellIndices< dimension >& index,
-            index_t direction ) const;
-
-        bool is_cell_on_border(
-            const GridCellIndices< dimension >& cell_indices ) const;
-
         index_t nb_vertices_in_direction( index_t direction ) const;
 

```

Comment:
what are grid vertex? cell center or cell corner?

Reply:
cell corners, the vertices needed to define the cell, like in any mesh